src/go/k8s/tests/e2e/additional-cmdline-arguments/01-assert.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: cluster
+status:
+  replicas: 1
+  restarting: false
+  conditions:
+    - type: ClusterConfigured
+      status: "True"
```

Comment:
Not in the full scope of this e2e test but it would be better if we could confirm if the `redpanda` command includes the new flags and not just if the container was launched. I think if we can confirm that it does apply the flags correctly that should be enough for now.

Reply:
I'd expect this to be covered in the rest of the testing suite. See here for example: https://github.com/redpanda-data/redpanda/pull/8401/files#diff-0cfeba1779bdecfec8476b8602a73be2ab7de5892aec5463b2866bc66f4b42b2R805